packages/plugin-react-swc/src/index.ts

```diff
@@ -141,6 +142,13 @@ const react = (_options?: Options): PluginOption[] => {
             '[vite:react-swc] The MDX plugin should be placed before this plugin',
           )
         }
+
+        // Suggest to use vite-plugin-react-oxc if `rolldown-vite` is used and no swc plugins are set
+        if ('rolldownVersion' in vite && !config.plugins.length) {
+          console.warn(
+            '[vite:react-swc] We recommend switching to `vite-plugin-react-oxc` for improved performance as no swc plugins are used. More information at https://vite.dev/rolldown',
```

Comment:
I'm not 100% sure about performances. In dev mode, transforming ts is very rarely the bottleneck and in build mode esbuild is used and also very rarely the bottleneck (it's source map generation for me).
But the nice benefit is one less dependency and better integrated with Vite

Reply:
Indeed, I think the big benefit (besides a speed-up during build for some apps) is the unification in terms of dependencies. With `rolldown-vite`, we ideally want to enable people using solely oxc (no esbuild, as mentioned in the migration guide and also in [the blog post](https://voidzero.dev/posts/announcing-rolldown-vite#esbuild-as-optional-dependency)).